static/etcher-rpm.repo

```diff
@@ -0,0 +1,7 @@
+#etcher-rpm - packages by  from Etcher
```

Comment:
Two spaces in "packages by  from Etcher", and also one of the by & from needs removing